LayoutTests/imported/w3c/web-platform-tests/screen-orientation/resources/sandboxed-iframe-locking.html

```diff
@@ -20,12 +20,7 @@
   }
 
   screen.orientation.unlock();
-  try {
-    await document.exitFullscreen();
-  } catch (error) {
-    data.result = "errored";
-    data.name = "fullscreen exit failed";
-  }
+  await document.exitFullscreen();
```

Comment:
Oh, whoops! This is a regression (probably my fault 😰) . The try/catch is needed here, otherwise it TIMEOUTs believe.

Reply:
This needs to be fixed upstream. Then reimported in this patch

Reply:
Fixing via https://github.com/web-platform-tests/wpt/pull/37301.